routes/routes.js

```diff
@@ -0,0 +1,39 @@
+const express = require('express')
+const router = express.Router()
+
+const passport = require('../helpers/ppInformation')
+const isLoggedIn = require('../helpers/loginBlock')
+const HomeController = require('../controllers/homeController')
+const AuthController = require('../controllers/authController')
+const Announcement = require('../models/announcement')
+
+
+// Main Page
+router.get('/', HomeController.index)
+
+// Tracker Page
+router.get('/home', isLoggedIn, HomeController.home)
+
+router.post('/create', HomeController.create)
```

Comment:
you should check all `post`, `put`, and `delete` routes too if the current user is authenticated or not too